templates/deployment-scheduler.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: scheduler
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: scheduler
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: scheduler
+        image: eanylin/airflow-helm:1.8.1-1.6.2
+        volumeMounts:
+          - name: dags
+            mountPath: /usr/local/airflow/dags
+        env:
+        - name: AIRFLOW_HOME
+          value: "/usr/local/airflow"
+        args: ["scheduler", "-n", "5"]
+      volumes:
+      - name: dags
+        hostPath:
+          path: /home/ubuntu/workbench/dags
```

Comment:
This directory (`/home/ubuntu/workbench/dags`) should be configurable from values.yaml and we may want to be using a PVC for this, so that something can inject dag files into it external to this chart.  Otherwise, it will be difficult to get the dag .py files into this directory, whatever it is, and know which host this will land on.